.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module"
+  },
+  env: {
+    browser: true,
+    es6: true
+  },
+  extends: ["airbnb-base"],
+  globals: {},
+  plugins: ["jest", "svelte3"],
+  rules: {
+    "import/no-extraneous-dependencies": ["error", { devDependencies: true }],
+    "no-param-reassign": [
+      "error",
+      { props: true, ignorePropertyModificationsFor: ["draft"] }
```

Comment:
With the store changes that were made recently that I think I saw, I'm not sure we need this any longer?